engine/graphics/src/null/graphics_null.cpp

```diff
@@ -1324,12 +1324,9 @@ namespace dmGraphics
         return StoreAssetInContainer(context->m_AssetHandleContainer, tex, ASSET_TYPE_TEXTURE);
     }
 
-    static int DoDeleteTexture(void* _context, void* _texture)
+    static int DoDeleteTexture(void* _h_texture, void* _texture)
     {
-        NullContext* context = (NullContext*)_context;
-        HTexture texture = (HTexture) _texture;
-        Texture* tex = GetAssetFromContainer<Texture>(context->m_AssetHandleContainer, texture);
-
+        Texture* tex = (Texture*) _texture;
```

Comment:
We don't touch the asset handle container on the texture thread any longer. We do that in the complete function